src/unix/linux_like/linux/musl/mod.rs

```diff
@@ -644,6 +641,20 @@ pub const PRIO_PROCESS: ::c_int = 0;
 pub const PRIO_PGRP: ::c_int = 1;
 pub const PRIO_USER: ::c_int = 2;
 
+cfg_if! {
+    if #[cfg(target_arch = "s390x")] {
+        pub const POSIX_FADV_DONTNEED: ::c_int = 6;
+        pub const POSIX_FADV_NOREUSE: ::c_int = 7;
+
+        pub const PTRACE_DETACH: ::c_uint = 17;
```

Comment:
`PTRACE_DETACH` is defined to the same value regardless of branch.

Reply:
Note that it's `c_uint` as you defined previously: https://github.com/rust-lang/libc/blob/5fd6540db9b14fae02ff8a11a6102ee99965ae30/src/unix/linux_like/linux/musl/b64/s390x.rs#L358

Reply:
I think this should actually be `::c_int` to be aligned with the `ptrace()` definition.  I used a simple awk script to generate the definitions based on the musl headers.